Makefile

```diff
@@ -0,0 +1,43 @@
+SHELL := /bin/bash
+
+.PHONY: help prepare-demo
+.DEFAULT_GOAL := help
+
+help: ## 💬 This help message :)
+	@grep -E '[a-zA-Z_-]+:.*?## .*$$' $(firstword $(MAKEFILE_LIST)) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+lint: ## 🔍 Lint the code base (but don't fix)
+	@echo -e "\e[34m$@\e[0m" || true
+	@./lint.sh
+
+lint-fix: ## 🌟 Lint and fix the code base
+	@echo -e "\e[34m$@\e[0m" || true
+	@./lint.sh -f
+
+build: ## 🔨 Build the Banking Application
+	@echo -e "\e[34m$@\e[0m" || true
+	@cd banking-app && npm run build
+
+test: ## 🧪 Test the Banking Application
+	@echo -e "\e[34m$@\e[0m" || true
+	@cd banking-app && ./test.sh
+
+start-host: build ## 🏃 Start the CCF Sandbox
+	@echo -e "\e[34m$@\e[0m" || true
+	@cd banking-app && /opt/ccf/bin/sandbox.sh --js-app-bundle ./dist/ --initial-member-count 3 --initial-user-count 0
+
+prepare-demo: ## 🚀 Prepare the banking application demo
+	@echo -e "\e[34m$@\e[0m" || true
+	@cd banking-app && ./prepare_demo.sh
```

Comment:
I think this make file is better than `prepare_demo.sh`. So how about just removing `prepare_demo.sh` and `prepare-demo`?

Reply:
I have removed prepare_demo and moved the copy of "constitution" file into the makefile.

We now `make start-host` and `make demo`